docs/dev-guide/contributing/merging.rst

```diff
@@ -4,20 +4,24 @@ Merging
 Pull Requests
 -------------
 
-You have some commits in a branch, and you're ready to merge. The Pulp Team makes
-use of pull requests for all but the most trivial contributions. Please have a
-look at our :doc:`Contribution checklist <index>`.
+You have some commits in a branch, and you're ready to merge. The Pulp Team
+makes use of pull requests for all contributions. Please have a look at our
+:doc:`Contribution checklist <index>`.
 
 On the GitHub page for the repo where your development branch lives, there will be
 a "Pull Request" button. Click it. From there you will choose the source and
 destination branches.
 
-If there is a bugzilla issue, please title the pull request "<bz_id> -
-Short Message". In the comment section below, please include a link to the
-issue. Use of GitHub's markdown for the link is prefered. Example:
-``[BZ-123456](http://link.tobug)`` Additionally, please also include a link to the
-pull request in the bugzilla comments.
+If there is a Bugzilla bug associated with the commit, please title the pull
+request "<bz_id> - Short Message". In the comment section below, please include
+a link to the issue. Use of GitHub's markdown for the link is prefered.
+Example: ``[BZ-123456](http://link.tobug)`` Additionally, please also include a
+link to the pull request in the bugzilla comments.
```

Comment:
I wonder if we should go ahead and change this to reference Redmine instead of Bugzilla. Since this is on master, we will probably be on Redmine long before this gets released.